src/components/MemberUpdate/MemberUpdate.tsx

```diff
@@ -73,7 +73,7 @@ export const MemberUpdate = ({
         if (res.success) {
             setAlertMessage({
                 title: `Modifications enregistrées`,
-                message: `Mise à jour des missions ok`,
+                message: `Mise à jour des missions ok.`,
```

Comment:
Histoire d'harmoniser un peu les messages
```suggestion
                message: `Les missions ont été mises à jour.`,
```